dbatools.psd1

```diff
@@ -67,6 +67,7 @@
 	
 	# Functions to export from this module
 	FunctionsToExport = @(
+		'Add-DbaToolsToProfile',

```

Comment:
💪 I am ;) And this is indeed an internal function. In the future when you do add an external/user function, I'll add it to the psd1. Reason being, it always causes conflicts so having one designated person resolves that issue.